carbon-design-system/carbon
# Update Elements packages to v11 layout

This PR updates our elements packages to the new v11 layout. In general, this means moving `scss/modules` files to the top-level `scss` directory and updating imports and tests accordingly. This PR also updates the migration doc with any discrepancies picked up along the way.

#### Changelog

**New**

**Changed**

- Update the following packages to the new layout
  - Colors
  - Elements
  - Grid
  - Layout
  - Motion
  - Themes
  - Type

**Removed**

#### Testing / Reviewing

- Verify tests pass as expected
- Verify there are no more `modules` folders or references to folders in sass files
- Verify styles still work as expected
- Verify elements previews still work as expected
